plugins/pulp_rpm/plugins/importers/yum/repomd/updateinfo.py

```diff
@@ -106,6 +106,13 @@ def _parse_collection(element):
     ret = {
         'packages': map(_parse_package, element.findall('package')),
     }
+
+    # a module element indicates this erratum being part of a module
+    modulemds = element.findall('module')
+    # a last-item-override logic applies
+    if modulemds:
+        ret['module'] = _parse_module(modulemds[-1])
```

Comment:
I suggest to fail the import of an erratum which contains multiple modules in a collection.
Otherwise it's very hard to notice this error in an erratum format + there is no guarantee that the last element is a correct one.

Reply:
OK, we should probably update our XML "schema" specification to explicitly state this limitation